9-palidrome-number.js

```diff
@@ -0,0 +1,41 @@
+// Previously, I did a string-based reversal in my coderbyte-challenge repo, so thought I'd mix things
+// up a bit and try the number-based reversal. This was def harder.
+// basically, here just pick out digits one by one, reverse the #, but don't convert it to a string.
+
+var isPalindrome = function(x) {
+  // check if # is less than 0
+  // I'm more used to converting to strings and then create an array, so this was tricky to
+  // process it as a reversal. Dunno, may be I was overthinking it? ended up looking at how to do this
+  // part on stackoverflow & MDN
+
+  const isNegative = x < 0 ? true : false;
+
+  // if it is, return false
+  if (isNegative) {
```

Comment:
Just a nit. I think if `isNegative` is only used for this purpose, then you could just do 
```js
if (x < 0) return false;
```

However I do like that the first case you solve/address is the an exit early type of edge case.

Reply:
ok, thanks!